### PR TITLE
Update tutorial_expressivity_fourier_series.py

### PR DESCRIPTION
**Pull request by co9olguy**
Some final bits of polish before final release & advertising.

@mariaschuld do you have a picture you recommend in for the metadata?

### CODE REVIEW

File: demonstrations/tutorial_expressivity_fourier_series.py [hunk 1]
```diff
@@ -1,15 +1,19 @@
-"""
+r"""
 Quantum models as Fourier series
 ================================
 
+.. meta::
+    :property="og:description": The class of functions a quantum model can learn is characterized by the structure of its Fourier series.
+    :property="og:image": ...
```
[the remark this answers is not in the document]
Comment (josh146) [reply demonstrations/tutorial_expressivity_fourier_series.py:7]: @mariaschuld feel free to use any image in the demo; simply go to the live page (https://pennylane.ai/qml/demos/tutorial_expressivity_fourier_series.html), right click any image and copy the image location

Comment (mariaschuld) [inline demonstrations/tutorial_expressivity_fourier_series.py:7]:
```suggestion
    :property="og:image": https://pennylane.ai/qml/_images/scheme.png
```

File: demonstrations/tutorial_expressivity_fourier_series.py [hunk 2]
```diff
@@ -382,7 +380,7 @@ def cost(weights, x, y):
 plt.scatter(x, target_y, facecolor='white', edgecolor='black')
 plt.plot(x, predictions, c='blue')
 plt.ylim(-1,1)
-plt.show()
+plt.show();
```
Comment (mariaschuld) [inline demonstrations/tutorial_expressivity_fourier_series.py:383]: Oh interesting, what is the difference? Does the ";" not introduce errors in the python file?

**Review by mariaschuld — APPROVED** [review]
Great, thanks Nathan. I added the image link! We could alternatively use the thumb of the demo itself...

File: demonstrations/tutorial_expressivity_fourier_series.py [hunk 3]
```diff
@@ -1,15 +1,19 @@
-"""
+r"""
 Quantum models as Fourier series
 ================================
 
+.. meta::
+    :property="og:description": The class of functions a quantum model can learn is characterized by the structure of its Fourier series.
```
Comment (mariaschuld) [inline demonstrations/tutorial_expressivity_fourier_series.py:6]:
```suggestion
    :property="og:description": The class of functions a quantum model can learn is expressed in terms of partial Fourier series.
```

File: demonstrations/tutorial_expressivity_fourier_series.py (hunk 2 above)
Comment (co9olguy) [reply demonstrations/tutorial_expressivity_fourier_series.py:383]: in some displays, `plt.show()` will give you a line like `<matplotlib figure at 0xDEADBEEF>` in addition to the picture

**Pull request merged**